### PR TITLE
Add warning if trigger scaling changes in request

### PR DESCRIPTION
**Pull request by samaloney**
Add new keyword `DATAWARN` if `0` ok if `1` check comments.
Add new comment indicating the trigger values maybe incorrect
Add test

### CODE REVIEW

File: stixcore/data/test/products/solo_LB_stix-21-6-24_0000000000-9999999999_V02_2402021788-59493.fits
Comment (samaloney) [inline stixcore/data/test/products/solo_LB_stix-21-6-24_0000000000-9999999999_V02_2402021788-59493.fits:1]: I'm not sure what all the fields mean or if they all need to be accurate?

File: stixcore/data/test/publish/rid_lut.csv
Comment (samaloney) [inline stixcore/data/test/publish/rid_lut.csv:1]: I'm not sure what all the fields mean or if they all need to be accurate? Same or update file below.

**Comment by samaloney** [comment]
I didn't change anything but I guess a release of the config repo was made? 

```
   Headers contain differences:
     Headers have different number of cards:
      a: 72
      b: 73
     Extra keyword 'VERS_CFG' in b: 'v0.1.3'
```

**Review by nicHoch — APPROVED** [review]
this is looking good

the error in the end2end is the new added header keyword

**Pull request merged**